### DISCUSSION TITLE
Passaggio nuovo anno - alunni

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#747] [Richieste di aiuto] [answered]
Salve, volevo chiedere un'info riguardo il passaggio di classe al momento del cambio d'anno.
Ad esempio lo studente X, se promosso, iscritto nell'A.S. 2023/24 nella classe 3N al momento del passaggio al nuovo anno risulterà in 4N automaticamente o bisogna effettuare manualmente il passaggio ?
@trinko

**Comment by Davide031** [comment]
Ho già effettuato il passaggio, è tutto automatico ma tu devi reinserire i parametri di inizio anno fine, quadrimestri ecc. E devi reinserire le classi

**Comment by trinko** [comment, marked as answer]
> Salve, volevo chiedere un'info riguardo il passaggio di classe al momento del cambio d'anno. Ad esempio lo studente X, se promosso, iscritto nell'A.S. 2023/24 nella classe 3N al momento del passaggio al nuovo anno risulterà in 4N automaticamente o bisogna effettuare manualmente il passaggio ? @trinko

Si, come ti hanno già risposto,  è automatico ma devi fare attenzione a:
- se ci sono nuove classi, le devi inserire prima di  effettuare il passaggio d'anno (es. esiste la 3N ma non la 4N, i promossi risulteranno senza classe),,

Le modifiche ai quadrimestri e alle date di inizio/fine le farai dopo aver effettuato il passaggio d'anno.

**Reply by luigiadoc** [reply]
Grazie a entrambi !
